foundation/normalize/README.md

```diff
@@ -0,0 +1,11 @@
+# Normalize
```

Comment:
Could do... same for the JS too I suppose. I'm just copying and pasting atm. Is this something to discuss later if we consider providing any tooling to generate docs/styleguides etc?